src/main/java/echo/Storage.java

```diff
@@ -0,0 +1,114 @@
+package echo;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+/**
+ * The Storage class handles loading tasks from a file and saving tasks to a file.
+ */
+public class Storage {
+    String filePath;
+
+    /**
+     * Constructs a Storage object with the specified file path.
+     *
+     * @param filePath The path of the file to load and save tasks.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+
+    /**
+     * Loads the content of the file specified by the file path and prints each line.
+     *
+     * @throws FileNotFoundException If the file does not exist.
+     */
+    public void loadTest() throws FileNotFoundException {
+        File file = new File(filePath);
+        Scanner saveScanner = new Scanner(file);
+        while (saveScanner.hasNextLine()){
+            System.out.println(saveScanner.nextLine());
+        }
+    }
+
+
+    /**
+     * Loads tasks from the file specified by the file path and returns them as a list.
+     *
+     * @return A list of tasks loaded from the file.
+     * @throws IOException If an I/O error occurs while reading the file.
+     */
+    public List<Task> load() throws IOException {
+        List<Task> tasks = new ArrayList<>();
+        File file = new File(filePath); //always using same file
+
+        if (file.exists()) {
+            Scanner saveScanner = new Scanner(file);
+            while (saveScanner.hasNextLine()) {
+                String input = saveScanner.nextLine();
+                String[] parts = input.split("\\|");
+
+                String type = parts[0];
+                String marked = parts[1];
+                String des = parts[2];
+                String info = parts[3];
+                switch (type) {
+                    case "T":
```

Comment:
Is this a coding standard violation? case statement should be in the same indentation as switch statement.